.github/actions/setup-builder/action.yaml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This was copied from arrow-rs , from https://github.com/apache/arrow-rs/pull/1510